src/_version_check.sh

```diff
@@ -125,11 +124,16 @@ function version_check() {
                 unset 'upstream_tags[i]'
             fi
         done
+        new_array=()
         for i in "${!upstream_tags[@]}"; do
             new_array+=( "${upstream_tags[i]}" )
         done
-        upstream_tags=("${new_array[@]}")
-        unset new_array
+        if "${#new_array[@]}" > 0; then
+                upstream_tags=("${new_array[@]}")
+                unset new_array
+        else
+                upstream_tags=("")
+        fi
```

Comment:
This checks if the array length is greater than 0 and then attempts to use it. Otherwise it sets it to an expected value of ""